woocommerce/google-listings-and-ads
# Get started page UX improvements: add customer quotes card

### Changes proposed in this Pull Request:

Part of #1392, this PR adds the customer quotes card to the new get started page.

Design for desktop: eQ9O4m2flzrcAiDMXkOW0m-fi-7350%3A121231
Design for mobile: eQ9O4m2flzrcAiDMXkOW0m-fi-7476%3A123025

### Screenshots:

#### Desktop

<img width="1257" alt="Screenshot 2022-04-27 at 18 47 38" src="https://user-images.githubusercontent.com/914406/165501942-c48ed499-4991-46ca-b5c3-418a6f795346.png">

#### Mobile

<img width="1254" alt="Screenshot 2022-04-27 at 18 48 29" src="https://user-images.githubusercontent.com/914406/165502080-fa649c4a-99d3-4f35-9e5d-902ec7adad70.png">

### Detailed test instructions:

1. Go to GLA's get started page: `https://domain.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fstart`
2. Check the fourth card of the get started page, the UI should align with the design.


### Changelog entry

> Update - Add the customer quotes card for UX improvements on get started page.
